aries-site/src/pages/components/card.mdx

```diff
@@ -69,34 +147,132 @@ A card by nature has limited real-estate, so it should present summarized inform
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/EventPromotionCard.js"
   docs="https://v2.grommet.io/card?theme=hpe#props"
   figma="https://www.figma.com/file/je5AYzMavRt1cG7rmw8j1W/HPE-Card-Component?node-id=1%3A2"
   grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Card/Card.js"
-  horizontalLayout
 >
-  <CallToActionCard />
+  <EventPromotionCard />
 </Example>
 
-#### Anatomy
+##### Event promotion (row-orientation)
 
-Call-to-action cards should be visually prominent as a means of attracting user attention.
+Depending on where a card is presented in a layout, it may be more appropriate for its content to be displayed in a row.
 
-TBD
+<Example
+  componentName="Card"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/EventPromotionRowCard.js"
+  docs="https://v2.grommet.io/card?theme=hpe#props"
+  figma="https://www.figma.com/file/je5AYzMavRt1cG7rmw8j1W/HPE-Card-Component?node-id=1%3A2"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Card/Card.js"
+>
+  <EventPromotionRowCard />
+</Example>
 
-#### Presentation
+##### Feature update
 
-- Call-to-action cards may be presented in a group with other call-to-action cards but also may be presented on their own.
-- When presented in a group, their layout/treatment can be independent from the layout/treatment of the surrounding call-to-action cards.
+Announcements about new offerings and features should excite current and potential customers about HPE's services. The use of this brand-approved gradient allows this card to stand out from its surrounding content.
 
-- Cards that are highly promotional should use elements that create higher visual emphasis (colorful brand-approved images, background colors/textures, etc.).
-  - Use cases (ordered from highest visual emphasis to lowest) include event promotion, new feature release, trial promotion.
-- Cards that are less promotional should still have enticing titles but do not necessarily need as strong of visual emphasis. The white brand-approved textures are appropriate if a subtle visual emphasis is desired.
-  - A use case is a software upgrade announcement.
+<Example
+  componentName="Card"
+  code="https://raw.githubusercontent.com/grommet/hpe-design-system/master/aries-site/src/examples/components/card/FeaturePromotionCard.js"
+  docs="https://v2.grommet.io/card?theme=hpe#props"
+  figma="https://www.figma.com/file/je5AYzMavRt1cG7rmw8j1W/HPE-Card-Component?node-id=1%3A2"
+  grommetSource="https://github.com/grommet/grommet/blob/master/src/js/components/Card/Card.js"
+>
+  <FeaturePromotionCard />
+</Example>
+
+##### Trial promotion (background image)
+
+The use of this brand-approved texture subtley differentiates this card from surrounding content.
+
+<Example
```

Comment:
This example doesn't look good when the site is in dark mode. I wonder if we should add a note reminding people to ensure the text has enough contrast when using text over a background image and that extra caution should be used if their site allows switching between light and dark mode

<img width="455" alt="Screen Shot 2022-09-27 at 1 30 46 PM" src="https://user-images.githubusercontent.com/54560994/192618842-c1888493-3050-4e66-8537-f0b91a02e34b.png">

Reply:
One thing to note is that setting `dark: true|false` explicitly when using a background image will prevent this sort of behavior, where the text color changes but the background doesn't.

Reply:
Thank you, will adjust.

Reply:
Done.